Form/Type/MediaType.php

```diff
@@ -77,14 +76,34 @@ public function buildView(FormView $view, FormInterface $form, array $options)
 
     /**
      * {@inheritdoc}
+     *
+     * @deprecated To be removed when bumping to Symfony 2.7+, use MediaType::configureOptions() instead
      */
     public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
+        if ($resolver instanceof OptionsResolver) {
```

Comment:
That isn't the condition being checked. Hint for ```setDefaultOptions()``` is ```OptionsResolverInterface```. So, I could call this method with my own implementation of ```OptionsResolverInterface```. Do you agree?